rgdc/rgdc/version.py

```diff
@@ -0,0 +1 @@
+__version__ = '0.1.0'
```

Comment:
Moving forward, should we have this version match RGD's version?

Reply:
I'd say so yes, but until we have a good feature match, we'll want to keep the client versions to prereleases, so that adding additional features don't misalign the versions.